platform/engineering/scheduled-downtime-process.md

```diff
@@ -0,0 +1,10 @@
+# Scheduled Downtime Process for VA.gov
+
+## Overview
+Any site upgrades, migration, or changes that affect will resolute in the public site to experience downtime, should follow the process below in order to allow the right parties to be aware, give enough time for other teams to make any preparations, and to minimize any effect to the Veterans. 
+
+## Steps Needed to be Taken to Schedule Downtime On VA.gov
+
+- Any changes that will result in downtime for the public website need to be communicated with your DEPO lead 2 Weeks before any planned downtime.
+- An announcement at the weekly Team of Teams meeting about the scheduled downtime needs to be made one week in advance of the downtime.
```

Comment:
Add additional bullet before last bullet:
- An announcement should be made at the same time in #vfs-all-teams, detailing the dates and times approved by the head of platform